connector/src/test/java/it/eng/connector/integration/ApplicationPropertyIntegrationTest.java

```diff
@@ -0,0 +1,91 @@
+package it.eng.connector.integration;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.util.UUID;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithUserDetails;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+
+import it.eng.connector.util.TestUtil;
+import it.eng.tools.model.ApplicationProperty;
+import it.eng.tools.model.IConstants;
+import it.eng.tools.model.Serializer;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class ApplicationPropertyIntegrationTest {

```

Comment:
Could you please replace this with extends BaseIntegrationTest, we already have it there, and to be consistent :)

Reply:
I can't find BaseIntegrationTest in the dependencies. What is the package??

Reply:
Done